src/gui/titlebar.cpp

```diff
@@ -0,0 +1,808 @@
+			continue;
+		}
+
+		if (iequals(setting_str, "version=detailed")) {
+			check_double_setting(TitlebarConfig::Setting::Version);
+			config.version = TitlebarConfig::VersionDisplay::Detailed;
+			continue;
+		}
+
+		if (iequals(setting_str, "cycles=on")) {
+			check_double_setting(TitlebarConfig::Setting::Cycles);
+			config.show_cycles = true;
+			continue;
+		}
+
+		if (iequals(setting_str, "cycles=off")) {
+			check_double_setting(TitlebarConfig::Setting::Cycles);
+			config.show_cycles = false;
+			continue;
+		}
+
+		if (iequals(setting_str, "mouse=none")) {
+			check_double_setting(TitlebarConfig::Setting::Mouse);
+			config.mouse = TitlebarConfig::MouseHintDisplay::None;
+			continue;
+		}
+
+		if (iequals(setting_str, "mouse=short")) {
+			check_double_setting(TitlebarConfig::Setting::Mouse);
+			config.mouse = TitlebarConfig::MouseHintDisplay::Short;
+			continue;
+		}
+
+		if (iequals(setting_str, "mouse=full")) {
+			check_double_setting(TitlebarConfig::Setting::Mouse);
+			config.mouse = TitlebarConfig::MouseHintDisplay::Full;
+			continue;
+		}
+
+		LOG_WARNING("SDL: Invalid 'window_titlebar' setting '%s', ignoring",
+		            setting_str.c_str());
+		config_needs_sync = true;
+	}
+
+	config_needs_sync |= !is_already_warned.empty();
+	if (config_needs_sync) {
+		sync_config();
+	}
+}
+
+void TITLEBAR_ReadConfig(const Section_prop& secprop)
+{
+	parse_config(secprop.Get_string("window_titlebar"));
+
+	GFX_RefreshTitle();
+}
+
+void TITLEBAR_AddConfig(Section_prop& secprop)
+{
+	constexpr auto always = Property::Changeable::Always;
+
+	Prop_string* prop_str = nullptr;
+
+	prop_str = secprop.Add_string("window_titlebar",
+	                              always,
+	                              "program=name dosbox=auto cycles=on mouse=full");
+	prop_str->Set_help(
+	        "Space separated list of information to be displayed on the window titlebar\n"
+	        "('program=name dosbox=auto cycles=on mouse=full' by default). If a parameter\n"
+	        "is not specified specified, its default value is used.\n"
+	        "Possible information to display are:\n"
+	        "  animation=<value>:  If set to 'on' (default), animate the audio/video\n"
+	        "                      recording mark. Set to 'off' to disable animation; this\n"
+	        "                      is usefull if your screen font produces weird results.\n"
+	        "  program=<value>:    Display the name of a running program.\n"
+	        "                      <value> can be one of:\n"
+	        "                        none:     Do not display program name.\n"
+	        "                        name:     Program name, with file extension (default).\n"
+	        "                        path:     Name, extension, and full absolute path.\n"
+	        "                        segment:  Displays program memory segment name.\n"
+	        "                        'Title':  Custom game title. Alternatively, you can use\n"
```

Comment:
Maybe just say "Custom name."

Name because "program name" was used so far, not "title". And you can use it for any program, not just games 😄

Reply:
OK, changed.